modules/installation/proc-installing-with-helm-cli.adoc

```diff
@@ -0,0 +1,56 @@
+[id="installing-with-helm-cli_{context}"]
+= Installing {product} with Helm CLI
+
+You can use the Helm CLI to install {product} on OpenShift or Kubernetes.
+
+.Prerequisites
+* An `oc` or `kubectl` session with a user that has the `developer` or `admin` role on an OpenShift or Kubernetes cluster.
+* Helm CLI.
+
+.Procedure
+. For a Kubernetes cluster, you  must first add the OpenShift Helm Charts repository. On Openshift, this is already present. 
++
+----
+$ helm repo add openshift-helm-charts https://charts.openshift.io/
+----
+
+. Create and activate the _<rhdh>_ OpenShift project or Kubernetes namespace:
++
+[subs="quotes+"]
+----
+$ NAMESPACE=_<rhdh>_
+$ oc new-project ${NAMESPACE} || oc project ${NAMESPACE}
+----
```

Comment:
@nickboldt is there a reason why we would document this for generic k8s instead of just supported OpenShift, since this is a RH downstream doc set, not upstream?

Reply:
yes, because we state support for various k8s flavours in the 1.1 and 1.2 docs (AKS, EKS). 

https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.1/html-single/release_notes_for_red_hat_developer_hub_1.1/index#supported-platforms
(simple table)

https://dxp-docp-prod.apps.ext-waf.spoke.prod.us-west-2.aws.paas.redhat.com/documentation/en-us/red_hat_developer_hub/1.2/html-single/release_notes_for_red_hat_developer_hub_1.2/index?lb_target=preview&check_logged_in=1#support-for-elastic-kubernetes-services-eks
https://dxp-docp-prod.apps.ext-waf.spoke.prod.us-west-2.aws.paas.redhat.com/documentation/en-us/red_hat_developer_hub/1.2/html-single/release_notes_for_red_hat_developer_hub_1.2/index?lb_target=preview&check_logged_in=1#support-for-elastic-kubernetes-services-eks
(multiple verbose paragraphs)

https://dxp-docp-prod.apps.ext-waf.spoke.prod.us-west-2.aws.paas.redhat.com/documentation/en-us/red_hat_developer_hub/1.2/html-single/administration_guide_for_red_hat_developer_hub/index?lb_target=preview&check_logged_in=1#assembly-rhdh-integration-aws_admin-rhdh
https://dxp-docp-prod.apps.ext-waf.spoke.prod.us-west-2.aws.paas.redhat.com/documentation/en-us/red_hat_developer_hub/1.2/html-single/administration_guide_for_red_hat_developer_hub/index?lb_target=preview&check_logged_in=1#assembly-rhdh-integration-aks_admin-rhdh